From foo@baz Thu Aug 20 12:12:31 PDT 2009
Date: Thu, 20 Aug 2009 12:12:31 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: hv: remove volatile usage from HvSynicApi.h

It's pretty pointless as no one is using this structure, but even so
the use of volatile is so wrong here it's sad...

Cc: Hank Janssen <hjanssen@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/hv/include/HvSynicApi.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/staging/hv/include/HvSynicApi.h
+++ b/drivers/staging/hv/include/HvSynicApi.h
@@ -212,7 +212,7 @@ struct hv_message {
 
 /* Define the synthetic interrupt message page layout. */
 struct hv_message_page {
-	volatile struct hv_message SintMessage[HV_SYNIC_SINT_COUNT];
+	struct hv_message SintMessage[HV_SYNIC_SINT_COUNT];
 };
 
 /* Define the synthetic interrupt controller event flags format. */
@@ -223,7 +223,7 @@ union hv_synic_event_flags {
 
 /* Define the synthetic interrupt flags page layout. */
 struct hv_synic_event_flags_page {
-	volatile union hv_synic_event_flags SintEventFlags[HV_SYNIC_SINT_COUNT];
+	union hv_synic_event_flags SintEventFlags[HV_SYNIC_SINT_COUNT];
 };
 
 /* Define SynIC control register. */
